pulseguardian/web.py

```diff
@@ -350,38 +350,78 @@ def update_info():
     pulse_username = request.form['pulse-user']
     new_password = request.form['new-password']
     password_verification = request.form['new-password-verification']
+    new_owner_list = _clean_owners_str(request.form['owners-list'])
```

Comment:
This isn't actually a list. It should be called either `new_owner_set` or just `new_owners`. Although actually, to differentiate from `new_owners` below, maybe this should be `new_owners_emails` or something like that.